.github/workflows/pull-request.yml

```diff
@@ -131,7 +131,7 @@ jobs:
   docker_build_cross_compile:
     name: "Docker build x-compile for aarch64"
     runs-on: ubuntu-latest
-    timeout-minutes: 20
+    timeout-minutes: 45
```

Comment:
Over 20 min to build a single image, that's a lot of time :thinking:

Reply:
Agree, but I guess this is the x-compilation price